fuse/new_orleans_so.py

```diff
@@ -10,28 +10,86 @@
 sys.path.append('../')
 
 
-def fuse_events(cprr, post):
+def fuse_events(cprr19, cprr20, post):
     builder = events.Builder()
-    builder.extract_events(cprr, {
+    builder.extract_events(cprr19, {
         events.COMPLAINT_RECEIVE: {
-            'prefix': 'receive', 'parse_date': True, 'keep': ['uid', 'agency', 'complaint_uid'],
+            'prefix': 'receive',
+            'parse_date': True,
+            'keep': ['uid', 'agency', 'complaint_uid'],
         },
         events.INVESTIGATION_START: {
-            'prefix': 'investigation_start', 'parse_date': True, 'keep': ['uid', 'agency', 'complaint_uid'],
+            'prefix': 'investigation_start',
+            'parse_date': True,
+            'keep': ['uid', 'agency', 'complaint_uid'],
         },
         events.INVESTIGATION_COMPLETE: {
-            'prefix': 'investigation_complete', 'parse_date': True, 'keep': ['uid', 'agency', 'complaint_uid'],
+            'prefix': 'investigation_complete',
+            'parse_date': True,
+            'keep': ['uid', 'agency', 'complaint_uid'],
+        },
+        events.OFFICER_ARRESTED: {
+            'prefix': 'arrest',
+            'parse_date': True,
+            'keep': ['uid', 'agency', 'complaint_uid'],
+        },
+        events.OFFICER_SUSPENDED: {
+            'prefix': 'suspension',
+            'parse_date': True,
+            'keep': ['uid', 'agency', 'complaint_uid'],
+        },
+        events.OFFICER_RESIGNED: {
```

Comment:
Should probably use `OFFICER_LEFT` instead. And you can add a `left_reason` column to the event table.

Reply:
Ok.